src/changelog/1.4.0/54-thread-context.xml

```diff
@@ -0,0 +1,8 @@
+<entry xmlns="http://logging.apache.org/log4j/changelog"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="http://logging.apache.org/log4j/changelog https://logging.apache.org/log4j/changelog-0.1.2.xsd"
+       type="changed">
+  <issue id="54" link="https://github.com/apache/logging-log4j-kotlin/issues/54"/>
+  <author id="github:rocketraman"/>
```

Comment:
```suggestion
  <issue id="54" link="https://github.com/apache/logging-log4j-kotlin/issues/54"/>
```

Reply:
Oops forgot this one due to a commit re-order. Fixed.